main.js

```diff
@@ -0,0 +1,33 @@
+
+$("button").on("click", function () {
+    let address = $("#addr-input").val()
```

Comment:
maybe make something like this into an objects. It will make your code a lot easier to read and can then be put into handlebars a lot better. 

`const appartement = { address: $("#addr-input").val(), minPrice: $("#min-p-input").val(), maxPrice: $("#max-p-input").val(), ... }